addOns/custompayloads/custompayloads.gradle.kts

```diff
@@ -0,0 +1,11 @@
+version = "1"
```

Comment:
Probably better to start using semantic version, as other add-ons will depend on this one.

Reply:
Fixed